elements/lisk-p2p/src/p2p.ts

```diff
@@ -375,6 +377,22 @@ export class P2P extends EventEmitter {
 				const wsPort: number = parseInt(queryObject.wsPort, BASE_10_RADIX);
 				const peerId = constructPeerId(socket.remoteAddress, wsPort);
 
+				// Check blacklist to avoid incoming connections from backlisted ips
```

Comment:
Maybe we can move this check before all the checks ([p2p.ts#L342](https://github.com/LiskHQ/lisk-sdk/blob/63e2b41f8a1073642ac6c8941e23825329cdccb4/elements/lisk-p2p/src/p2p.ts#L342)) related to handshake and return early if its a blacklisted ip?